parler_tts/modeling_parler_tts.py

```diff
@@ -2518,6 +3409,81 @@ def generate(
                 f" {generation_config.max_length}. This can lead to unexpected behavior. You should consider"
                 " increasing `max_new_tokens`."
             )
+        
+        use_dynamic_cache_by_default = False
+        if generation_config.cache_implementation is not None and model_kwargs.get("past_key_values") is not None:
```

Comment:
It's worth noting that in the current implementation, the class `ParlerForConditionalGeneration` (encoder-decoder) supports static cache through `.generate`, but not the class `ParlerForCausalLM` (decoder-only). For the latter to support this, we would need to add these lines of logic to `ParlerForCausalLM.generate`.

Reply:
Yes indeed, I discussed that with @ylacombe that explained to me that the generate of `ParlerForCausalLM` is not updated anyway. Such a thing could be done when integrating Parler-TTS to Transformers

Reply:
Sounds good to me!